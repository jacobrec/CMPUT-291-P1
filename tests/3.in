1
testy
password
3
10000
3
0
3
16
3
17
7
2

