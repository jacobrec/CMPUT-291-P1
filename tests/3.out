Please sign in to use App:
1) login
2) exit
Username: Password: Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
Registration Number: No registration found
Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
Registration Number: No registration found
Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
Registration Number: │ Registration Number  │     First Name     │     Last Name      │    Expiry Date     │
╞══════════════════════╪════════════════════╪════════════════════╪════════════════════╡
│          16          │       Speedy       │    McSpeedFace     │     2020-11-02     │
Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
Registration Number: │ Registration Number  │     First Name     │     Last Name      │    Expiry Date     │
╞══════════════════════╪════════════════════╪════════════════════╪════════════════════╡
│          17          │       Speedy       │    McSpeedFace     │     2021-08-08     │
Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
Please sign in to use App:
1) login
2) exit
